components/CalendarEvent.js

```diff
@@ -128,15 +128,8 @@ const CalendarEventBase = (props) => {
                 onPress={() => {
                     if (event?.status === 'BOOKED' || event?.status === 'WAITING') {
```

Comment:
Maybe we can remove this condition snd let user enter reservation details regardless of status. Since we have edit button, we can use it to control which reservation can allow further edit.

Reply:
Okay. If status == confirmed || seated, should "Send SMS" button disable?